web/src/features/hfiCalculator/components/WeeklyViewTable.tsx

```diff
@@ -109,7 +119,7 @@ export const WeeklyViewTable = (props: Props): JSX.Element => {
                 </TableCell>
               </TableRow>
               {Object.entries(centre.planning_areas)
-                .sort((a, b) => (a[1].name < b[1].name ? -1 : 1))
+                .sort((a, b) => (a[1].name.slice(-3) < b[1].name.slice(-3) ? -1 : 1)) // sort by zone code
```

Comment:
it's possible, yes, but I don't think it wouldn't be a trivial refactor because the `PlanningArea` object is used in a bunch of places in our codebase

Reply:
Maybe for now just make a small helper function called `getZoneFromAreaName` that does this so it's self documenting and reusable.